content/post/articles/how_to_create_a_hotspot_on_linux.md

```diff
@@ -0,0 +1,153 @@
+---
+title: "How to create a hotspot on Linux 🐧 ?"
+date: 2022-02-21T17:16:14+01:00
+images: []
+categories: [tutorial, network, linux]
+tags: [network, linux]
+authors: [pythonbrad]
+---
+
+{{<figure src="/how_to_create_a_hotspot_on_linux/hotspot.png" alt="Hotspot">}}
+
+# Table of Contents
+1. [What is a hotspot ?](#what-is-a-hotspot-)
+2. [How to create it ?](#how-to-create-it-)
+3. [Requirements](#requirements)
+    1. [Hostapd](#hostapd)
+    2. [Dhcpd](#dhcpd)
+    3. [Wifi card](#a-wifi-card-with-ap-mode-support-is-needed)
+4. [Implementation phase](#implementation-phase)
+    1. [Simple configuration](#80211bgn-with-wpa2-psk-and-ccmp)
```

Comment:
You used "sample" instead of "single". Can you consolidate the two? The table of contents and the content? Thank you.

Reply:
I will finish the redaction before the next request for review.